tests/features/licenseexport_cdx.feature

```diff
@@ -36,38 +36,158 @@ Scenario: User Downloads license information for CycloneDX SBOM from SBOM Explor
+	# Test data https://drive.google.com/drive/folders/1Z6y6gMegutBeUuc_8LkpYxKeGz_KYG9H?usp=drive_link
+	# sbom - tc_1730_license_escape.json package - pkg:rpm/rhel/annobin@12.31-2.el9?arch=x86_64&upstream=annobin-12.31-2.el9.src.rpm&distro=rhel-9.4
+	Given User is on SBOM license information file
+	When User selects a package with Single license expression with cpe information
+	Then name column should contain the value of SBOM name from metadata.component.name field from SBOM json
+	And namespace column should be empty
+	And group column should contain the value of metadata.component.group field from SBOM json
+	And version column should contain the value of metadata.component.version field from SBOM json
+	And package reference column should contain the value of components.purl from SBOM json
+	And license column should be empty
+	And license name column should be empty
+	And license expression column should contain the value of whole license expression in a single row from components.license.expression field from SBOM json
+	And alternate package reference column should contain the value of CPE from components.cpe field from SBOM json
+
+Scenario: Verify the license information for a package on the CycloneDX SBOM with multiple license ids
+	# Test data https://drive.google.com/drive/folders/1Z6y6gMegutBeUuc_8LkpYxKeGz_KYG9H?usp=drive_link
+	# sbom - cdx_sbom.json package - pkg:maven/jakarta.el/jakarta.el-api@3.0.3?type=jar
+	Given User is on SBOM license information file
+	When User selects a package with multiple license sections
+	Then The report should have multiple rows for the same package corresponding to each license section
+	And Values on columns name, namespace, group, version, package reference, license name, license expression and alternate package reference should be same for all the rows
+	And Column license id for each row should contain the value from the components.license.id field of the corresponding license section
+
+Scenario: Verify the license information for a package on the CycloneDX SBOM with multiple license names
+	# Test data https://drive.google.com/drive/folders/1Z6y6gMegutBeUuc_8LkpYxKeGz_KYG9H?usp=drive_link
+	# sbom - cdx_sbom.json package - pkg:maven/xpp3/xpp3_min@1.1.4c?type=jar
+	Given User is on SBOM license information file
+	When User selects a package with multiple license sections
+	Then The report should have multiple rows for the same package corresponding to each license section
+	And Values on columns name, namespace, group, version, package reference, license id, license expression and alternate package reference should be same for all the rows
+	And Column license name for each row should contain the value from the components.license.name field of the corresponding license section
+
+Scenario: Verify the license information for a package on the CycloneDX SBOM with license id and license name
```

Comment:
@mrrajan from the scenario name it is not clear here if we are talking about license name and license id columns only or is it containing multiple license names or multiple license ids like the previous scenario

Reply:
@ikanias it is multiple section validation - but instead of multiple license id or license name, it has license id and license name under the section.